lib/vanagon/platform/windows.rb

```diff
@@ -1,6 +1,7 @@
 class Vanagon
   class Platform
     class Windows < Vanagon::Platform
+      attr_accessor :proj_root
```

Comment:
@mckern @McdonaldSeanp 
One *minor* problem with using Root_Dir accessed from Project.
How do I access the Project object/instance from within windows.rb ?